src/Avalonia.Controls/Presenters/TextPresenter.cs

```diff
@@ -110,7 +110,8 @@ public class TextPresenter : Control
 
         static TextPresenter()
         {
-            AffectsRender<TextPresenter>(CaretBrushProperty, SelectionBrushProperty);
+            AffectsRender<TextPresenter>(CaretBrushProperty, SelectionBrushProperty, 
+                TextElement.FontFamilyProperty, TextElement.FontSizeProperty, TextElement.FontStyleProperty, TextElement.FontWeightProperty, TextElement.FontStretchProperty, TextElement.ForegroundProperty);
```

Comment:
This is kind-of a nit, but if parameters are so long they span multiple lines there should be one on each line. This really helps with readability in this case.

```csharp
AffectsRender<TextPresenter>(
    CaretBrushProperty,
    SelectionBrushProperty, 
    TextElement.FontFamilyProperty,
    TextElement.FontSizeProperty,
    TextElement.FontStyleProperty,
    TextElement.FontWeightProperty,
    TextElement.FontStretchProperty,
    TextElement.ForegroundProperty);
```

Reply:
Yep, fixed that